<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>M_SetPulse</class>
 <widget class="QWidget" name="M_SetPulse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>303</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>367</width>
    <height>303</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>367</width>
    <height>303</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>脉冲</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>369</width>
     <height>305</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_all_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>50</y>
      <width>357</width>
      <height>212</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>7</y>
       <width>69</width>
       <height>195</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QCheckBox" name="checkBox_path0_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>14</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path 0</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_path1_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>53</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path 1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_path2_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>93</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path 2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_path3_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>132</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path 3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_path4_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>173</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path 4</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_path0_2">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>8</y>
       <width>271</width>
       <height>35</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Width：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_0_width_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>us</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_0_delay_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>us</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox__path1_2">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>49</y>
       <width>271</width>
       <height>35</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Width：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_1_width_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>us</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_1_delay_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>us</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_path2_2">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>90</y>
       <width>271</width>
       <height>35</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Width：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2_width_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>us</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2_delay_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>us</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_path3_2">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>130</y>
       <width>271</width>
       <height>35</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Width：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3_width_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>us</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3_delay_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>us</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_path4_2">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>170</y>
       <width>271</width>
       <height>35</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Width：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4_width_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>us</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4_delay_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_41">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>us</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_42">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>356</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_period_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>13</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_pulse_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>12</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Pulse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>14</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Period：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>us</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_default_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>266</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>默认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonapply">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>266</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ok_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>266</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
